src/__tests__/commands/utilities/ping.test.ts

```
@@ -0,0 +1,395 @@
+describe('Ping Command', () => {
+	let mockInteraction: Partial<ChatInputCommandInteraction>;
+	let mockSentMessage: any;
+	let mockClient: any;
+
+	beforeEach(() => {
+		// Reset all mocks
+		vi.clearAllMocks();
+
+		// Create mock sent message
+		mockSentMessage = {
+            createdTimestamp: 1640995800, // Mock timestamp (milliseconds semantics for test diffing)
```

ESLint

[error] Expected indentation of 3 tabs but found 12 spaces.

(indent)